packages/manager/modules/netapp/src/dashboard/network-configuration/template.html

```diff
@@ -29,15 +29,15 @@
         >
             <oui-field label="{{:: $ctrl.LABELS.VRACK }}" size="xl">
                 <oui-select
-                    data-items="$ctrl.vracks"
-                    data-match="internalName"
+                    data-items="$ctrl.availableVracks"
+                    data-match="vrack.internalName"
                     data-model="$ctrl.selectedVrack"
                     data-searchable="true"
                     data-placeholder="{{:: 'netapp_network_configuration_vrack_field_placeholder' | translate }}"
                     data-disabled="$ctrl.disableVrackField"
                     name="vrack"
                 >
-                    <span data-ng-bind="$item.internalName"></span>
+                    <span data-ng-bind="$item.vrack.internalName"></span>
```

Comment:
I think this is unnecessary as you set `data-match="vrack.internalName"`, ui-kit should display `$item.vrack.internalName` automatically